tests/test_shield_display.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+import time
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@pytest.mark.firefox_preferences({'privacy.trackingprotection.enabled': True})
+def test_tracking_protection_shield_displays(foxpuppet, selenium):
+    """Tests if the tracking protection icon displays"""
+    browser = foxpuppet.browser
+    assert not browser.navbar.is_tracking_shield_displayed
+    selenium.get('https://www.washingtonpost.com/')
```

Comment:
shavar tests currently use a handful of external sites to verify tracking protection.  normally this is considered bad practice as we're relying on an external source that we don't have control over.  However, it may simply be that we accept that risk because we want real data.  I'll file an issue to determine what we want to do and we can decide later